lib/pr_checker.js

```diff
@@ -164,6 +172,11 @@ class PRChecker {
       return false;
     }
 
+    function isFast(label) {
+      isFastTrack = label === 'fast-track';
```

Comment:
Why do we need a closure here? All the checks can be done in the function passed to `labels.some()`

Reply:
@joyeecheung To check afterwards if the label was fast-track and log the message out.

Reply:
@cPhost But now that we don't log that in the messages...we don't really need this right? All we need is

```js
const fast = labels.some((l) => (['code-and-learn', 'fast-track'].includes(l)) ||
  (labels.length === 1 && labels[0].name === 'doc');
if (fast) {
  cli.info(...)
  return true;
}
```

Reply:
Oh i though we do not count `code-and-learn` fixing it right now.

Reply:
done.